data/json/materials.json

```diff
@@ -906,5 +882,25 @@
       { "fuel": 1, "smoke": 1, "burn": 2 },
       { "fuel": 1, "smoke": 1, "burn": 5 }
     ]
+  },
+  {
+    "type": "material",
+    "ident": "feces",
+    "name": "Feces",
```

Comment:
@kevingranade: Are you okay with that?

Reply:
Yea, the issue has always been with player elimination being pointless to depict, not that I want to insist shit doesn't exist.